source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,410 @@
+.. _minio-k8s-deploy-operator-helm:
+
+=========================
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequisites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing cluster using Kubernetes 1.19.0 or later (1.21.0 or later recommended).
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Helm <https://helm.sh/docs/intro/install/>`__ version 3.8 or greater.
+* `yq <https://github.com/mikefarah/yq/#install>`__ version 4.18.1 or greater.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
+
+This procedure assumes familiarity the with referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kubernetes Documentation <>`.
+
+
+Procedure
+---------
+
+
+Install Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Download the Helm charts
+
+   On your local host, download the Operator and Tenant Helm charts to a convenient directory:
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/operator-|operator-version-stable|.tgz
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/tenant-|operator-version-stable|.tgz
+
+   Each chart contains a ``values.yaml`` file you can customize to suit your needs.
+   For example, you may wish to change the MinIO root user credentials or the Tenant name.   
+   For more information about the format of this file, see `Helm Charts <https://helm.sh/docs/topics/charts/>`__.
+  
+#. Deploy Operator
+
+   The following Helm command deploys the MinIO Operator using the downloaded chart:
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      helm install \
+      --namespace minio-operator \
+      --create-namespace \
+      minio-operator operator-|operator-version-stable|.tgz
+
+#. Configure the MinIO deployment
```

Comment:
This isn't the MinIO deployment, right? We're creating services for the Operator pods, Operator console pods, and the console JWT secret.

Reply:
Ah yes. That might conceivably be relevant to the Tenant section, but not here. Perhaps "Configure Operator" would be better. I'll bring this up when we talk next week, hopefully I'll be more enlightened about this entire yaml section after that.